packages/integration-tests/cmr.js

```diff
@@ -105,8 +106,8 @@ async function waitForConceptExistsOutcome(cmrLink, expectation, retries = 3, in
         timeout: interval * retries
       }
     );
-
-    console.log(`waitForConceptExistsOutcome() succeeded. conceptExists('${cmrLink}') === ${expectation}`);
+    const shardSettleTimeMs = 1000;
+    await sleep(shardSettleTimeMs);
```

Comment:
To make this easier for "Marc in 12 months" to understand, it may be helpful to extract this sleep out into a function called `waitForCmrConsistency()` or something similar.  You could also add a comment here, but a comment is more likely to go stale.